frameit/lib/frameit/config_parser.rb

```diff
@@ -80,8 +80,8 @@ def validate_values(values)
           when 'color'
             UI.user_error!("Invalid color '#{value}'. Must be valid Hex #123123") unless value.include?("#")
           when 'padding'
-            unless value.kind_of?(Integer) || value.split('x').length == 2
-              UI.user_error!("padding must be type integer or pair of integers of format 'AxB'")
+            unless value.kind_of?(Integer) || value.split('x').length == 2 || (value.end_with?('%') && value.to_f > 0)
+              UI.user_error! "padding must be type integer or pair of integers of format 'AxB' or a percentage of screen size"
```

Comment:
Can we keep the parenthesis in the call to `user_error!`? I realize this is valid, but it does not match the style of the rest of the file.

Also, can you add parens on the call to `user_error!` on line 87 below (in the `font_scale_factor` code)?